### ISSUE TITLE
Index all texlive.* packages

### ISSUE THREAD

**Issue by Ekleog** [NixOS/nixos-search#442]
AFAIU we have almost all CTAN packages packaged already. However, they don't display in the package search page.

I'm thinking it'd make sense to index all texlive.* packages and not only texlive.combined.* ones. Unfortunately, I can't find anything when I grep for haskellPackages in this repo, which does appear in the search page.

Would such a change make sense to you?

**Comment by ysndr** [comment]
The file that is (going to) be used for this is defined as part of nixpkgs: https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/packages-config.nix

You can make a PR there, once #419 is merged, we will list nixpkgs according to that file.
Texlive is huge though, I don't exactly know about possible implications of including that group.

**Comment by Ekleog** [comment]
Interesting thanks! Do you happen to know the command that can be used to regenerate the index locally, so that I can check the compile time and compiled size don't blow up? (and also add it as a comment on top of the file you linked so it can be found easier thereafter)

**Comment by ysndr** [comment]
Just implemented (not yet merged in #445).

You can use 
```
$ nix run github:nixos/nixos-search/cli/nixpkgs-archive-import -- --json nixpkgs-archive <path-to-archive or folder> 
```

Sadly, you wont see it on the official frontend.
you could set up your own elasticsearch locally and configure the push operation and frontend accordingly or grep the json output (I'm sorry) for it.

**Comment by ncfavier** [comment]
`github:nixos/nixos-search/cli/nixpkgs-archive-import#flake-info` *

**Comment by tannerlegvold** [comment]
Heads up its looks like this got [merged into main](https://github.com/NixOS/nixos-search/commit/d2ca8bb5c97a0a7c0ecdb5849a48c12e88ec19ac) two years ago. So this issue should be closed.

**Comment by ncfavier** [comment]
This issue is about indexing texlive packages. Texlive packages are still not indexed.
